contracts/usul/BaseStrategy.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "../interfaces/IFractalUsul.sol";
+import "./IBaseStrategy.sol";
+import "@gnosis.pm/zodiac/contracts/factory/FactoryFriendly.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+
+abstract contract BaseStrategy is
+    OwnableUpgradeable,
```

Comment:
same with this, can this just be `Ownable`?

Reply:
Same as above